rust/phoenix-sdk-core/src/sdk_client_core.rs

```diff
@@ -261,117 +306,81 @@ impl SDKClientCore {
         market_key: &Pubkey,
         quote_atoms: u64,
     ) -> Result<f64> {
-        let market = self
+        let metadata = self
             .markets
             .get(market_key)
             .ok_or_else(|| anyhow!("Market not found! Please load in the market first."))?;
-        Ok(quote_atoms as f64 / market.quote_multiplier as f64)
-    }
-
-    /// Takes in a number of quote atoms, converts to floating point number of whole tokens, and prints it as a human readable string to the console
-    pub fn print_quote_amount(&self, market_key: &Pubkey, quote_amount: u64) -> Result<()> {
-        let market = self
-            .markets
-            .get(market_key)
-            .ok_or_else(|| anyhow!("Market not found! Please load in the market first."))?;
-        println!(
-            "{}",
-            get_decimal_string(quote_amount, market.quote_decimals)
-        );
-        Ok(())
-    }
-
-    /// Takes in a number of base atoms, converts to floating point number of whole tokens, and prints it as a human readable string to the console
-    pub fn print_base_amount(&self, market_key: &Pubkey, base_amount: u64) -> Result<()> {
-        let market = self
-            .markets
-            .get(market_key)
-            .ok_or_else(|| anyhow!("Market not found! Please load in the market first."))?;
-        println!("{}", get_decimal_string(base_amount, market.base_decimals));
-        Ok(())
+        Ok(quote_atoms as f64 / metadata.quote_atoms_per_quote_unit as f64)
     }
 
     /// Takes in information from a fill event and converts it into the equivalent quote amount
-    pub fn fill_event_to_quote_amount(&self, market_key: &Pubkey, fill: &Fill) -> Result<u64> {
+    pub fn fill_event_to_quote_atoms(&self, market_key: &Pubkey, fill: &Fill) -> Result<u64> {
         let &Fill {
             base_lots_filled: base_lots,
             price_in_ticks,
             ..
         } = fill;
-        self.order_to_quote_amount(market_key, base_lots, price_in_ticks)
+        self.order_to_quote_atoms(market_key, base_lots, price_in_ticks)
     }
 
-    /// Takes in tick price and base lots of an order converts it into the equivalent quote amount
-    pub fn order_to_quote_amount(
+    /// Takes in tick price and base lots of an order converts it into the equivalent quote atoms
+    pub fn order_to_quote_atoms(
```

Comment:
Don't love this function name. Should represent the number of quote atoms either to transact with the order (if offer) or is locked in the book (if bid)

Reply:
We should also refactor to either:
* actually take in some sort of order object and update doc accordingly OR
* update function name to show that this takes a base amount and a price